01-12 08:32:31 jdbc[2]: Connection not closed
java.lang.Exception: Stack Trace
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:95)
	at org.h2.Driver.connect(Driver.java:73)
	at java.sql.DriverManager.getConnection(DriverManager.java:582)
	at java.sql.DriverManager.getConnection(DriverManager.java:207)
	at com.thebrain.personal.a.a.r.b(vhd:1765)
	at com.thebrain.personal.a.a.r.A(vhd:1017)
	at com.thebrain.personal.a.a.r.a(vhd:907)
	at com.thebrain.personal.a.a.s.a(whd:326)
	at com.thebrain.personal.view.od.a(axc:2750)
	at com.thebrain.personal.view.od.a(axc:2368)
	at com.thebrain.personal.view.bc.run(axc:2492)
	at java.lang.Thread.run(Thread.java:680)
01-12 12:04:20 jdbc[3]: SQLException
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-140]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1345)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1323)
	at org.h2.jdbc.JdbcConnection.createStatement(JdbcConnection.java:165)
	at com.thebrain.common.b.u.a(aif:153)
	at com.thebrain.common.b.u.a(aif:78)
	at com.thebrain.common.b.u.a(aif:143)
	at com.thebrain.personal.a.a.r.c(vhd:593)
	at com.thebrain.c.e.s.F(uvf:369)
	at com.thebrain.personal.a.a.s.A(whd:338)
	at com.thebrain.personal.view.od.a(axc:3266)
	at com.thebrain.personal.controller.App.a(fkd:1034)
	at com.thebrain.personal.b.c.f.handleQuit(dcd:258)
	at com.apple.mrj.MRJPriv$MRJQuitApplicationAdapter.handleQuit(MRJPriv.java:416)
	at com.apple.eawt._AppEventLegacyHandler$7.dispatchEvent(_AppEventLegacyHandler.java:146)
	at com.apple.eawt._AppEventLegacyHandler.sendEventToEachListenerUntilHandled(_AppEventLegacyHandler.java:168)
	at com.apple.eawt._AppEventLegacyHandler.handleQuitRequestWith(_AppEventLegacyHandler.java:144)
	at com.apple.eawt._AppEventHandler$_QuitDispatcher.performUsing(_AppEventHandler.java:372)
	at com.apple.eawt._AppEventHandler$_QuitDispatcher.performUsing(_AppEventHandler.java:365)
	at com.apple.eawt._AppEventHandler$_AppEventDispatcher$1.run(_AppEventHandler.java:474)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:633)
	at com.thebrain.personal.c.d.b.dispatchEvent(xxc:9)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
01-13 09:47:20 jdbc[2]: SQLException
org.h2.jdbc.JdbcSQLException: General error: "java.lang.NullPointerException" [50000-140]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:156)
	at org.h2.message.DbException.convert(DbException.java:279)
	at org.h2.message.DbException.toSQLException(DbException.java:252)
	at org.h2.message.TraceObject.logAndConvert(TraceObject.java:386)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:190)
	at com.thebrain.common.b.u.c(aif:851)
	at com.thebrain.common.b.u.B(aif:588)
	at com.thebrain.common.model.g.B(jhf:100)
	at com.thebrain.personal.a.a.xb.a(mid:73)
	at com.thebrain.common.model.n.D(nhf:212)
	at com.thebrain.common.model.n.b(nhf:157)
	at com.thebrain.common.model.m.run(nhf:166)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.lang.NullPointerException
	at org.h2.jdbc.JdbcConnection.afterWriting(JdbcConnection.java:1363)
	at org.h2.jdbc.JdbcStatement.afterWriting(JdbcStatement.java:901)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:187)
	... 8 more
01-13 09:47:20 jdbc[2]: SQLException
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-140]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:327)
	at org.h2.message.DbException.get(DbException.java:167)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.message.DbException.get(DbException.java:133)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1345)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1323)
	at org.h2.jdbc.JdbcConnection.createStatement(JdbcConnection.java:165)
	at com.thebrain.common.b.u.a(aif:153)
	at com.thebrain.common.b.u.a(aif:78)
	at com.thebrain.common.b.u.a(aif:143)
	at com.thebrain.personal.a.a.r.c(vhd:593)
	at com.thebrain.c.e.s.F(uvf:369)
	at com.thebrain.personal.a.a.s.A(whd:338)
	at com.thebrain.personal.view.od.a(axc:3266)
	at com.thebrain.personal.view.od.a(axc:2323)
	at com.thebrain.personal.view.od.a(axc:2368)
	at com.thebrain.personal.view.bc.run(axc:2492)
	at java.lang.Thread.run(Thread.java:680)
